build.sh

```diff
@@ -49,7 +49,7 @@ EOF
     echo "BML_MPI              {yes,no}                    (default is ${BML_MPI})"
     echo "BML_TESTING          {yes,no}                    (default is ${BML_TESTING})"
     echo "BUILD_DIR            Path to build dir           (default is ${BUILD_DIR})"
-    echo "BLAS_VENDOR          {,Intel,MKL,ACML,GNU}       (default is '${BLAS_VENDOR}')"
+    echo "BLAS_VENDOR          {,Intel,MKL,ACML,GNU,Auto}  (default is '${BLAS_VENDOR}')"
```

Comment:
Isn't that what `BLAS_VENDOR=""` does already?

Reply:
It seems like, but I don't see the 
  set(BLAS_FOUND TRUE)
  set(LAPACK_FOUND TRUE)
lines at inside the = "" condition

Reply:
Ok, that's a good point. Basically you would like to skip all checks with `BLAS_VENDOR=Auto`, right?